plugins/com.google.cloud.tools.eclipse.integration.appengine/src/com/google/cloud/tools/eclipse/integration/appengine/BaseProjectTest.java

```diff
@@ -60,6 +65,11 @@ public void tearDown() {
         SwtBotProjectActions.deleteProject(bot, project.getName());
       } catch (TimeoutException ex) {
         // If this fails it shouldn't fail the test, which has already run
+        logger.log(Level.SEVERE, "Timeout deleting project: " + project.getName(), ex);
+        String fileName = SWTBotPreferences.SCREENSHOTS_DIR + "/" + "timeout-" + project.getName()
+            + "." + SWTBotPreferences.SCREENSHOT_FORMAT.toLowerCase();
+        SWTUtils.captureScreenshot(fileName);
```

Comment:
Where is the location locally on Travis, and can we download it from Travis?

Reply:
Travis only preserves the build log, unfortunately.  We'll need to upload any artefacts that we want to preserve (see #501).  Travis has support for uploading stuff directly to GCS but I haven't tried it (yet).